examples/landing/next.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  assetPrefix: process.env.NODE_ENV === 'production' ? '/examples/landing' : '/',
+  assetPrefix:
+    process.env.NODE_ENV === 'production' ? '/examples/landing' : '/',
```

Comment:
Maybe something's up with your prettier config haha, let's keep the current change?